frontend/wallet/src/render/components/TransactionsView.re

```diff
@@ -93,41 +107,65 @@ let extractTransactions: Js.t('a) => array(TransactionCell.Transaction.t) =
 [@react.component]
 let make = () => {
   let transactionQuery = Transactions.make(~publicKey="123", ());
+  let emptyTransactionsView = 
+    <div className=Styles.alertContainer>
+      <div className=Styles.noTransactionsAlert>
+        <Alert
+          kind=`Info
+          message={|Your Coda wallet is empty!
+            Once you've sent or received Coda, 
```

Comment:
We can probably drop the exclamation mark in the copy since theres already an exclamation mark in the icon, also you can't send Coda if you don't have any, so we should probably drop the "sent or" bit.
```suggestion
            Once you receive Coda, 
```